honeybee_3dm/face.py

```diff
@@ -0,0 +1,81 @@
+"""Create Honeybee face objects(Face, Shade, Aperture, Door) from planar geometries
+in a Rhino 3DM file."""
+
+import warnings
+
+# Importing core Honeybee & Ladybug Geometry dependencies
+from honeybee.face import Face
+from honeybee.shade import Shade
+from honeybee.aperture import Aperture
+from honeybee.door import Door
+from honeybee.facetype import face_types
+from honeybee.typing import clean_and_id_string, clean_string
+
+# Importing dependencies from Honeybee-3dm package
+from .togeometry import to_face3d
+from .helper import filter_objects_by_layers
+
+
+def import_faces(rhino3dm_file, tolerance=None):
+    """Import Rhino planar geometry as Honeybee faces.
+
+    This function looks up a rhino3dm file, converts the objects
+    on the layer name "roof", "wall", "floor", "airwall", "shade", and  "aperture"
+    to Honeybee objects, and converts them to Honeybee faces.
+
+    Args:
+        rhino3dm_file: A Rhino3DM file object.
+        tolerance: A number for model tolerance. By default the tolerance is set to
+            the ModelAbsoluteTolerance value in input 3DM file.
+
+    Returns:
+        A list of Honeybee faces.
+
+    """
+    hb_faces = []
+
+    # TODO: Add an input to customize layer names
```

Comment:
Ah. This will be so nice!

Reply:
Yes. We will have to implement this for the users' flexibility.